ui/mainwindow.cpp

```diff
@@ -115,31 +151,23 @@ MainWindow::MainWindow(QWidget *parent)
     , ui(new Ui::MainWindow)
     , m_stateManager(this)
 {
+
+    const auto styleOverride = gammarayStyleOverride();
+    if (styleOverride) {
+        applyStyle(styleOverride);
```

Comment:
Yeah, I wanted to keep this in line with what's done for the out-of-process mode. Plus the style changes are only local to the main window.